CHANGES.md

```diff
@@ -1,3 +1,9 @@
+### Unreleased
+  
+  * Added `ToField` instances for case-insensitive strict and lazy text.
+    Added `citext` and `_citext` to `Database.PostgreSQL.Simple.TypeInfo.Static`.
+    Thanks to Max Tagher for the implementation.
+
```

Comment:
Wasn't sure if you wanted contributors to make their own changes to the changelog

Reply:
Doesn't matter much to me one way or the other.  :)